### PR TITLE
Shapes edits

### PR DESCRIPTION
**Pull request by tabatkins**
Fixing up a few long-pending Shapes edits for #9053, #8695, #824, and a Guillaume complaint (about the basic shape grammar definitions not being in a prod) that I can't find the issue number for right now.

### CODE REVIEW

**Comment by cdoublev** [comment]
Will fix https://github.com/w3c/csswg-drafts/issues/7390, thanks!

File: css-shapes-1/Overview.bs [hunk 1]
```diff
@@ -423,106 +457,64 @@ Supported Shapes</h3>
...
-					the center of the circle.
-					This defaults to center if omitted.
-				</li>
-			</ul>
-		</dd>
-		<dt><dfn>ellipse()</dfn> =
-			ellipse( [ <<shape-radius>>{2} ]? [ at <<position>> ]? )
-		</dt>
+			* The <<radial-size>> argument defines
+				the circle's radius.
```
Comment (astearns) [inline css-shapes-1/Overview.bs:463]: Was there a reason for omitting “Negative values are invalid” here and for ellipse()?

**Review by astearns — APPROVED** [review]
One nit about removing prose for disallowing negative values (only a nit because it’s expressed in the syntax)

File: css-shapes-1/Overview.bs (hunk 1 above)
Comment (tabatkins) [reply css-shapes-1/Overview.bs:463]: Already defined as part of the production, which has bounds on the `<length-percentage>`. It didn't need to be reproduced.

**Pull request merged**